app/src/main/java/org/mozilla/fenix/collections/CreateCollectionFragment.kt

```diff
@@ -191,16 +191,24 @@ class CreateCollectionFragment : DialogFragment(), CoroutineScope {
     private fun handleBackPress(backPressFrom: SaveCollectionStep) {
         when (backPressFrom) {
             SaveCollectionStep.SelectTabs -> dismiss()
-            SaveCollectionStep.SelectCollection -> getManagedEmitter<CollectionCreationChange>().onNext(
-                CollectionCreationChange.StepChanged(SaveCollectionStep.SelectTabs)
-            )
+            SaveCollectionStep.SelectCollection -> {
+                if (viewModel.tabs.size > 1) {
+                    getManagedEmitter<CollectionCreationChange>().onNext(
+                        CollectionCreationChange.StepChanged(SaveCollectionStep.SelectTabs)
+                    )
+                } else dismiss()
+            }
```

Comment:
Nit: make this `dismiss()` use the `then` branch instead of the `else` branch, to match L202 below.